zerver/tests/test_subs.py

```diff
@@ -5006,7 +5016,7 @@ def test_multi_user_subscription(self) -> None:
         streams_to_sub = ["multi_user_stream"]
         with (
             self.capture_send_event_calls(expected_num_events=5) as events,
-            self.assert_database_query_count(39),
+            self.assert_database_query_count(41),
```

Comment:
Looks like it's +3 now? I think we can fix this as a follow-up, but let's make sure it's tracked as a priority on the groups board rather than hidden away here.

Reply:
Opened #32562